### PR TITLE
Update README for setup dev environment with docker

### PR DESCRIPTION
**Pull request by obregonia1**
Thank you for the wonderful conference the other day!

For now, I have written the minimum method to get the application up and running. Feedback would be appreciated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,31 @@ The app is built with Ruby on Rails 7. It uses `importmap-rails`, `turbo-rails`,
...
 See [`.env.sample`](.env.sample) for required/optional environment variables.
+
+## Setup dev environment with docker
+```bash
+$ cp .env.sample .env
+```
+
+Generate required environment variables
+```bash
+$ docker exec -it conference-app rails c
```
Comment (unasuke) [inline README.md:40]: `docker compose` looks more correct. What do you think?

```suggestion
$ docker compose exec -it conference-app rails c
```

Comment (obregonia1) [reply README.md]: @unasuke 
I see. Thanks for the feedback!

File: README.md [hunk 2]
```diff
@@ -29,3 +29,31 @@ The app is built with Ruby on Rails 7. It uses `importmap-rails`, `turbo-rails`,
...
+Copy to `.env`
+```bash
+VAPID_PUBLIC_KEY=YOUR_PUBLIC_KEY
+VAPID_PRIVATE_KEY=YOUR_PRIVATE_KEY
+```
+
+And then execute below
+```bash
+$ docker-compose up
+$ docker exec conference-app rails db:prepare
```
Comment (unasuke) [inline README.md:58]: 😉 

```suggestion
$ docker compose up
$ docker compose exec conference-app rails db:prepare
```

Comment (obregonia1) [reply README.md]: @unasuke 
🙏💦

**Comment by unasuke** [comment]
Thank you!

**Review by unasuke — APPROVED** [review]

**Pull request merged**